Electron/src/components/Sidebar/ModalAddSongPlaylist/Accordion/AddSongPlayListAccordion.tsx

```diff
@@ -131,7 +131,7 @@ export default function AddSongPlayListAccordion({
     setLoadingUploadSong(true);
     setIsCloseAllowed(false);
 
-    if (!formDataSong || !songFile) {
+    if (!songFile) {
       return;
     }
 
```

Comment:
still keeping songFile validation to make typescript happy cause its throw error that value undefined is not assignable to type blog  in payload "file"

Reply:
Yeah, `songFile` is undefined if no file is selected